scene/gui/color_picker.cpp

```diff
@@ -1531,6 +1654,97 @@ void ColorPicker::_pick_finished() {
 	picker_window->hide();
 }
 
+void ColorPicker::_update_menu_items() {
+	if (!options_menu) {
+		options_menu = memnew(PopupMenu);
+		add_child(options_menu);
```

Comment:
```suggestion
		add_child(options_menu, false, INTERNAL_MODE_FRONT);
```
All direct children should be internal.